packages/libs/react-ui/src/components/NavHeader/NavHeader.stories.children.tsx

```diff
@@ -0,0 +1,17 @@
+import { Button } from '@components/Button';
+import React, { FC } from 'react';
+
+export const NavHeaderChildren: FC = () => {
+  return (
+    <Button
+      as="button"
+      icon="Link"
+      onClick={() => {}}
+      style={{ marginLeft: '1rem' }}
```

Comment:
Can you move this to styles?

Reply:
This is just sample content. It's really irrelevant. I only added it so you can have some example content in the right hand side of the header. I could remove it all together perhaps?